pkg/products/rhssouser/reconciler.go

```diff
@@ -211,13 +218,105 @@ func (r *Reconciler) reconcileComponents(ctx context.Context, installation *inte
 	}
 	r.logger.Infof("The operation result for keycloak %s was %s", kc.Name, or)
 
+	host := r.Config.GetHost()
+
+	r.logger.Infof("Found host %s", host)
+	r.logger.Infof("Found host %s", host)
+	r.logger.Infof("Found host %s", host)
+	r.logger.Infof("Found host %s", host)
+	r.logger.Infof("Found host %s", host)
+	r.logger.Infof("Found host %s", host)
```

Comment:
I guess these^ were added for debugging? :)

Reply:
yes, removed :-)